test/ExpressiveInstallerTest/AddPackageTest.php

```diff
@@ -15,12 +17,8 @@ class AddPackageTest extends OptionalPackagesTestCase
 {
     /**
      * @dataProvider packageProvider
-     *
-     * @param string $packageName
-     * @param string $packageVersion
-     * @param null|int $expectedStability
      */
-    public function testAddPackage($packageName, $packageVersion, $expectedStability)
+    public function testAddPackage(string $packageName, string $packageVersion, ?int $expectedStability)
```

Comment:
Just to point out - this type hint is correct `?int`, we don't have default `null` value so the type must be nullable.